lzy/whiteboard/src/main/java/ai/lzy/whiteboard/model/Whiteboard.java

```diff
@@ -46,13 +32,12 @@ public boolean equals(Object o) {
             return false;
         }
         Whiteboard that = (Whiteboard) o;
-        return id.equals(that.id) && name.equals(that.name) && fields.equals(that.fields) && tags.equals(that.tags) &&
-               storage.equals(that.storage) && namespace.equals(that.namespace) && status == that.status;
+        return id.equals(that.id);
```

Comment:
With this change assertEquals will work incorrectly. We can create equalsAllFields method and use assertTrue(wb1.equalsAllFields(wb2)) instead.

Reply:
Do we really want equals to check only id?